entrypoint.py

```diff
@@ -19,8 +23,10 @@
 INFO = '\033[34m'
 NOTICE = '\033[33m'
 
-if 'RELEASE_TOKEN' not in os.environ:
-    print(ERROR + "RELEASE_TOKEN needs to be set in env. Exiting." + ENDC)
+deploy_key = os.environ.get('DEPLOY_KEY')
+if not deploy_key and 'RELEASE_TOKEN' not in os.environ:
+    print(f"{ERROR}Either RELEASE_TOKEN or DEPLOY_KEY needs to be set in env. "
+          f"Exiting.{ENDC}")
```

Comment:
I believe you can do a `if (X,Y) not in os.environ` here.

Like:

```python
a = [1,2,3,4]

if 1 or 5 in a:
  print("hello")
```

Reply:
`,` constructs tuples in Python, so `(X,Y) not in os.environ` will check whether the tuple `x, y` exists as key in `os.environ` and will always be false.

In your example, you are checking whether `1` is true or whether `5` is contained in `a`:

```pycon
>>> a = [2, 3, 4]
>>> 1 or 5 in a
1
>>> bool(1 or 5 in a)
True
>>> if 1 or 5 in a: print('whoops')
... 
whoops
```

What the code above does, maybe a bit too subtle: it sets the value of env var `DEPLOY_KEY` to the name `deploy_key`. If no such env var exists, the value will be `None`. It then checks whether `deploy_key` evaluates to false. That is the case for `None` as well as an empty string, so if you set the env var `DEPLOY_TOKEN`, but to an empty string, it will be caught here as well. I can change that to a bit more explicit code though if you like.

Reply:
I think that would be good. I expect we will have people who are not very python knowledgeable wanting to contribute (like me!) so more explicit is better.

Reply:
I don't much care as long as it's consistent within any particular grouping of things.